spec/models/move_spec.rb

```diff
@@ -219,8 +219,28 @@
     it { is_expected.to validate_presence_of(:reference) }
   end
 
+  describe '#nomis_event_id' do
```

Comment:
This will be removed after this gets merged: https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/747